vault-plugins/flant_iam/model/role.go

```diff
@@ -187,10 +192,6 @@ func (r *RoleRepository) Iter(action func(*Role) (bool, error)) error {
 }
 
 func (r *RoleRepository) Sync(objID string, data []byte) error {
-	if data == nil {
-		return r.Delete(objID)
-	}
-
```

Comment:
Why do we loose tombstone handling here? Is it impossible to get tombstone for a role? In other words, what if data is nil?

Reply:
New vision is to not delete anything, just make it  "archived", so there are no tomb stones in system